recipes/example/meta.yaml

```diff
@@ -21,6 +21,8 @@ source:
 build:
   number: 0
   # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for Python 2.7, 3.4, and above, for all major OSs.
+  # Add the line "skip: True  # [py < 34]" (for example) to limit to Python 3.4 and newer, or "skip: True  # [not win]" to limit to Windows.
```

Comment:
@jakirkham Yes, I originally used just `win` for the reasons you suggested, but my rationale here was:

- Most *users* are Windows users (this is at least true of scikit-image, but I suspect it holds for most of the Python stack)
- Devs (myself included) have traditionally neglected this audience

So I thought using this example would at least make devs pause. It also shows an extra feature (clause negation).